packages/vue/src/components/molecules/SfModal/SfModal.js

```diff
@@ -48,48 +52,24 @@ export default {
   },
   methods: {
     close() {
-      this.$emit("close");
-      // remove left code
-    },
-    keydownHandler(e) {
-      if (e.key === "Escape" || e.key === "Esc" || e.keyCode === 27) {
-        this.close();
-      }
-      if (this.visible) {
-        // TODO: this is an error right now. what shall it do?
-        //  document.addEventListener("keydown", keydownHandler);
-        this.$emit("close");
-      } else {
-        // TODO: this is an error right now. what shall it do?
-        //  document.removeEventListener("keydown", keydownHandler);
-        this.$emit("open");
-      }
+      this.$emit("toggle", !this.visible);
```

Comment:
on `close` method, we should always set visible to false, not?

```suggestion
      this.$emit("toggle", false);
```

Reply:
otherwise, imho the method should be named `toggle` instead of `close`, wdyt?